app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+class StaticPagesController < ApplicationController
+  def index
+    @pagy, @posts = pagy(Post.all, items: 5)
```

Comment:
```suggestion
    @pagy, @posts = pagy Post.all, items: 5
```
5 em đưa vào setting nhé
Viết thêm scope sắp xếp thay cho việc sử dụng all